oktapam/fwprovider/resource_server_checkout_settings.go

```diff
@@ -0,0 +1,277 @@
+	plan.IncludeList = updatedServerCheckoutSettings.IncludeList
+	plan.ExcludeList = updatedServerCheckoutSettings.ExcludeList
+
+	// Set state to fully populated data
+	diags = resp.State.Set(ctx, plan)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+}
+
+// Delete implements resource.Resource.
+func (r *serverCheckoutSettingsResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
+	// Retrieve the server checkout settings values from the plan
+	var state serverCheckoutSettingsResourceModel
+	diags := req.State.Get(ctx, &state)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+	defaultCheckoutDurationInSeconds := int32(900)
+	// Call the SDK client to reset the server checkout settings to default values
+	serverCheckoutSettings := &pam.ResourceCheckoutSettings{
+		CheckoutRequired:          false,
+		CheckoutDurationInSeconds: &defaultCheckoutDurationInSeconds,
+		IncludeList:               []string{},
+		ExcludeList:               []string{},
+	}
+
+	_, err := r.client.SDKClient.ProjectsAPI.UpdateResourceGroupServerBasedProjectCheckoutSettings(ctx, r.client.Team, state.ResourceGroup, state.Project).ResourceCheckoutSettings(*serverCheckoutSettings).Execute()
+
+	if err != nil {
+		resp.Diagnostics.AddError("Error updating server checkout settings", err.Error())
+		return
+	}
+
+	// Set state to empty data
+	state.Id = types.StringValue("")
+	diags = resp.State.Set(ctx, state)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+	// should we delete the state here?
+	// resp.State.RemoveResource(ctx)
+}
+
+// Configure adds the provider configured client to the resource.
+func (r *serverCheckoutSettingsResource) Configure(_ context.Context, req resource.ConfigureRequest, resp *resource.ConfigureResponse) {
+	// Add a nil check when handling ProviderData because Terraform
+	// sets that data after it calls the ConfigureProvider RPC.
+	if req.ProviderData == nil {
+		return
+	}
+
+	provider, ok := req.ProviderData.(*OktapamFrameworkProvider)
+
+	if !ok {
+		resp.Diagnostics.AddError(
+			"Unexpected Data Source Configure Type",
+			fmt.Sprintf("Expected *OktapamFrameworkProvider, got: %T. Please report this issue to the provider developers.", req.ProviderData),
+		)
+
+		return
+	}
+	r.client = provider.SDKClientWrapper
+}
+
+func formatServerCheckoutSettingsID(resourceGroupID string, projectID string) string {
+	// server checkout settings don't have an identifier in itself and is really an attribute of a project.
+	// we manage it as a separate resource since it's lifecycle is somewhat separate from a project.
+	// since project password settings are managed as a separate resource with format resourceGroupID/projectID already,
+	// we can just append the server_checkout_settings to the end of the ID
+	return fmt.Sprintf("%s/%s/server_checkout_settings", resourceGroupID, projectID)
```

Comment:
You don't need to add server_checkout_settings. For a project and resource, there can be only one server checkout settings.